gatling-http/src/main/scala/io/gatling/http/check/HttpCheck.scala

```diff
@@ -29,7 +30,7 @@ import io.gatling.http.response.{ Response, ResponseBodyUsageStrategy }
  * @param scope the part of the response this check targets
  * @param responseBodyUsageStrategy how this check uses the response body
  */
-case class HttpCheck(wrapped: Check[Response], scope: HttpCheckScope, responseBodyUsageStrategy: Option[ResponseBodyUsageStrategy])
+case class HttpCheck(wrapped: Check[Response], scope: Set[HttpCheckScope], responseBodyUsageStrategy: Set[ResponseBodyUsageStrategy])
```

Comment:
* scope: "should we add a default status check?", "can we discard body chunks because there's no check using the body?"
* body usage strategy: "how does this check use the body so we can compute in the most efficient manner this input from the response body chunks?"